cmd/soroban-cli/src/commands/config/identity/add.rs

```diff
@@ -10,14 +11,15 @@ pub enum Error {
 }
 
 #[derive(Debug, clap::Args)]
+#[group(skip)]
```

Comment:
pasting this here for visibility and indexing 

<img width="566" alt="looks like the named ArgGroup stuff is there mostly to enable future features; We use the same struct names ALL OVER THE PLACE so it would be quite a pain to give them all unique names; that's why the group(skip) stuff is needed" src="https://user-images.githubusercontent.com/221614/227234475-f8fe9cf7-ddd7-4754-938a-6d18ad467e06.png">